active/DateTimePicker/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,150 @@
+
+There is only one major API change that will come with these styling updates, which is adding a new class called `ColorFilterOverlayControl`. This class provides the logic behind displaying the selection rectangle that spans across all three rows in DatePicker and TimePicker, and giving it the correct background color. 
+
+# Visual Examples
+<!-- Use this section to provide a brief description of the feature.
+For an example, see the introduction to the PasswordBox control 
+(http://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box). -->
+
+**DatePicker:**
+
+As you can see on the left, the old version of DatePicker contains squared elements inside of it. The new version of DatePicker will round off these elements.
+
+![Shows the old DatePicker visuals alongside the new DatePicker visuals](images/datepicker-evolution.PNG)
+
+**TimePicker:**
+
+Similarly to DatePicker, the old version of TimePicker contains squared elements inside of it. The new version of TimePicker will round off these elements.
+
+![Shows the old TimePicker visuals alongside the new TimePicker visuals](images/timepicker-evolution.PNG)
+
+**Rounded backplates and focus visuals**
+
+As you can see in the mockup below, all buttons and items (individual cells) within DatePicker and TimePicker will receive a rounded grey backplate when hovered or pressed. In addition to this, columns within DatePicker and TimePicker will receive rounded focus rectangles (as shown below).
+
+![Shows a DatePicker with scroll and check buttons being hovered, as well as the last column being focused upon](images/DPTP-other-states.PNG)
+
+# API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+## New APIs
+
+`public Class ColorFilterOverlayControl`
+
+Represents a class that takes a rectangle and transforms into the coordinate space of a TargetElement, and then runs a ColorMatrixEffect on that region to select the alpha channel and replace with a ReplacementColor.
+
+`public UIElement TargetElement { get; set; }`
+
+This is a property of `ColorFilterOverlayControl`. Gets or sets the UIElement whose coordinate space should be replaced with a color. 
+
+`public Color ReplacementColor { get; set; }`
+
+This is a property of `ColorFilterOverlayControl`. Gets or sets the Color that should fill the target element. 
```

Comment:
```suggestion
This is a property of `ColorFilterOverlayControl`. Gets or sets the Color to use instead of the RGB channels of the TargetElement.
```